ghost/milestone-emails/lib/MilestonesEmailService.js

```diff
@@ -35,8 +35,11 @@ module.exports = class MilestonesEmailService {
     #repository;
 
     /**
-     * @type {ghostMailer} */
-    #mailer;
+     * @type {boolean} */
+    #slackEnabled;
```

Comment:
We should try to remove the concept of slack from this package, it doesn't need to know about it! Instead we can always emit an event, and the `slack-notification` service can decide whether or not it should do anything with it based on it's internal state